internal/policies/manager.go

```diff
@@ -161,7 +180,9 @@ func (m *Manager) ApplyPolicies(ctx context.Context, objectName string, isComput
 	g.Go(func() error {
 		return m.scripts.ApplyPolicy(ctx, objectName, isComputer, rules["scripts"], pols.SaveAssetsTo)
 	})
-	// TODO g.Go(func() error { return m.apparmor.ApplyPolicy(ctx, objectName, isComputer, rules["apparmor"]) })
+	g.Go(func() error {
```

Comment:
I think we need to filter the apparmor rule when there is no Ubuntu Pro subscription, though.